### DISCUSSION TITLE
Is there an equivalent of centeredDirectionalMove for Multitouch?

### DISCUSSION THREAD

**Discussion by smammy** [rxhanson/Multitouch-Community#173] [Q&A] [answered]
With Rectangle or Rectangle Pro I can set the hidden pref `centeredDirectionalMove` to `-int 2` to disable centering when using the "snap" movements:
```sh
defaults write com.knollsoft.Rectangle centeredDirectionalMove -int 2 # for Rectangle
defaults write com.knollsoft.Hookshot centeredDirectionalMove -int 2 # for Rectangle Pro
```

Is there an equivalent setting for Multitouch?

Thanks for the great apps.

**Comment by rxhanson** [comment, marked as answer]
```
defaults write com.brassmonkery.Multitouch centeredDirectionalMove -int 2
```
I haven't tested this out, but I think it should be the same.

**Reply by smammy** [reply]
It worked! Thanks.
